src/gui_main.cpp

```diff
@@ -294,17 +294,17 @@ void runUpdate(const std::string pathToAppImage) {
     }
 
     if (updater.hasError()) {
-        log("Update failed!");
+        log("Update failed");
         progressBar.selection_color(FL_RED);
         progressBar.redraw();
         Fl::check();
-        fl_alert("Update failed!");
+        fl_alert("Update failed");
         exit(0);
     } else {
         progressBar.selection_color(FL_GREEN);
         progressBar.redraw();
         Fl::check();
-        log("Successfully updated AppImage!");
+        log("Successfully updated");
```

Comment:
I'd rather write "Update successful" now, sounds better and is more consistent regarding the error messages.